app/views/Dashboard/CombinedIndicators/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { _cs } from '@togglecorp/fujs';
+import React, { useCallback } from 'react';
+import { ContainerCard, List } from '@the-deep/deep-ui';
+import MultiDataProgressBar from '#components/MultiDataProgressBar';
```

Comment:
add 1 space between library import and absolute import

Reply:
Done